### ISSUE TITLE
Apparent incompatibility with 6.5.0

### ISSUE THREAD

**Issue by STRML** [loganfsmyth/babel-plugin-transform-decorators-legacy#19]
I don't have a minimal test case yet, but this babelrc works pre-6.5.0, and throws this error with 6.5.0:

```
ModuleBuildError: Module build failed: SyntaxError: /project/ui/views/Component.jsx: Decorators are not supported yet in 6.x pending proposal update.
```

The .babelrc:

```
{
  "presets": [
    "es2015",
    "stage-0",
    "react",
    "async-to-bluebird"
  ],
  "plugins": [
  ]
}
```

Edit: Appears at this time that putting the plugin in the babelrc (and not in babel-loader's query) fixes the issue.

**Comment by sergeymakoveev** [comment]
+1

**Comment by loganfsmyth** [comment]
I can't seem to reproduce this. Would you be up for putting together a reproducible example repository?

To verify, you have no Babel config in your webpack config beyond the basic enabling of the loader?

**Comment by STRML** [comment]
https://github.com/STRML/babel-broken-legacy-decorators

**Comment by rufman** [comment]
I can confirm that I have experienced this issue as well

**Comment by loganfsmyth** [comment]
I've got a fix up in PR: https://github.com/babel/babel/pull/3348

**Comment by loganfsmyth** [comment]
This should now be fixed in babel-core@6.5.2.
